release.js

```diff
@@ -112,6 +115,7 @@ const main = async () => {
     console.log(chalk`{yellow Release cancelled by user}`);
   }
 
+  exec('git checkout main');
```

Comment:
I don't want a local `main` branch, the only thing it can do is get out of sync with origin.

```suggestion
  exec('git checkout origin/main');
```

Reply:
I actually removed this checkout line completely, and any checkout of a local `main` branch!